Assets/Xbox Live/Scripts/XboxLiveUserHelper.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+
+public class XboxLiveUserHelper : Singleton<XboxLiveUserHelper>
+{
+
+    public List<XboxLiveUserInfo> XboxLiveUsersInTheScene { get; private set; }
```

Comment:
>XboxLiveUsersInTheScene [](start = 34, length = 23)

is this don't destroy on load?  does it apply the current scene only or across all scenes?  if all scenes, odd var name.